qiita_ware/test/test_ebi.py

```diff
@@ -37,6 +43,18 @@ def setUp(self):
         self.sample3_fp = ebi_test_file('sample3.fastq.gz')
 
         self.temp_dir = gettempdir()
+        self.files_to_remove = []
+        self.ppd_to_remove = []
+
+    def tearDown(self):
+        for f in self.files_to_remove:
+            if isdir(f):
+                rmtree(f)
+            else:
+                remove(f)
+
+        for _id in self.ppd_to_remove:
```

Comment:
I don't think this is needed.

Actually, I just noticed that this test file is unsafe!!! Can you put the class decorator `@qiita_test_checker()` to the test class, so we avoid to run the test if this a production environment? That decorator will automatically drop the schema after each test, so you don't need to delete the processed data, only the files that it generates.